hr_attendance_autoclose/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'HR Attendance Auto Close',
+    'version': '11.0.1.0.0',
+    'category': 'Human Resources',
+    'summary': 'Close stale Attendances',
+    'website': 'https://github.com/OCA/hr',
+    'author': 'Odoo Community Association (OCA)',
```

Comment:
@aheficent can you please check author name it's right `'author': 'Eficent, Odoo Community Association (OCA)',` or put as it is.

Reply:
I think it is fine as it is as the copyright in the files. Thanks.